kombu/transport/redis.py

```diff
@@ -1229,7 +1230,7 @@ def _get_client(self):
                 global_keyprefix=self.global_keyprefix,
             )
 
-        return redis.StrictRedis
```

Comment:
Can you please elaborate on this change?

Reply:
The library has renamed `StrictRedis` to `Redis` but kept the old name for backwards compatibility, but someday they'll remove it.

https://github.com/redis/redis-py/blob/master/CHANGES#L580